tests/testing_data/inference.json

```diff
@@ -1,6 +1,10 @@
 {
     "dataset_dir": "/workspace/data/Task09_Spleen",
+    "import_glob": "$import glob",
     "device": "$torch.device('cuda:0' if torch.cuda.is_available() else 'cpu')",
+    "set_seed": "$monai.utils.set_determinism(0)",
+    "print_test_name": "$print('json_test')",
+    "print_glob_file": "$print(@import_glob.__file__)",
```

Comment:
I feel these 2 lines:
```
"import_glob": "$import glob",
"print_glob_file": "$print(@import_glob.__file__)",
```
is not simpler than:
```
"print_glob_file": "$print(__import__('glob').__file__)",
```

If we can set the imports outside this config file (for example in the `run` script), then the expression can be:
```
"print_glob_file": "$print(glob.__file__)",
```

What do you think?

Thanks.

Reply:
The local workaround wouldn't support the possibility of setting package variables.